### PR TITLE
Layout with fixed positions of header, sidebar, Toc.

### PR DESCRIPTION
**Pull request by postJScriptum**
Layout changes based on #48.

### CODE REVIEW

File: assets/scripts/modules/toc.js [hunk 1]
```diff
@@ -21,4 +21,9 @@ export function init() {
 		toc.classList.toggle( 'table-of-contents--hide' );
 		contentContainer.classList.toggle( 'content-container--no-cot-visible' );
 	} );
+
+	window.addEventListener( 'scroll', function() {
```
Comment (Reinmar) [inline assets/scripts/modules/toc.js]: Couldn't we use `position:sticky`? It's kind of progressive enhancement, so it doesn't have to work everywhere.

File: template/guide.pug [hunk 2]
```diff
@@ -22,6 +22,12 @@ block content
 							h5.table-of-contents__title Contents
 							+tableOfContents( tableOfContents )
 
+		div.main-footer
```
Comment (Reinmar) [inline template/guide.pug]: Do we have to move this piece? Perhaps we should define a different block in the base layout?

**Comment by Reinmar** [comment]
This PR is far from being ready.

When there's not enough space, the last items in the left menu (and I presume that ToC as well) are not accessible at all.

screen shot 2016-09-22 at 16 09 56

File: assets/scripts/modules/toc.js (hunk 1 above)
Comment (postJScriptum) [inline assets/scripts/modules/toc.js]: We can't becouse Internet Explorer doesn't support 'sticky' position type.

Comment (Reinmar) [inline assets/scripts/modules/toc.js]: Besides, this code is terrible for performance. If anything, this should use position:fixed once some condition is met.

**Comment by Reinmar** [comment]
And also this is broken:

screen shot 2016-09-22 at 16 19 05

File: assets/scripts/modules/toc.js (hunk 1 above)
Comment (Reinmar) [inline assets/scripts/modules/toc.js]: IE is not a problem if only this doesn't totally blow up there. It's a progressive enhancement for browsers which support pos:sticky.

**Comment by postJScriptum** [comment]
I think that all issues has been resolved:
1. Fix on extra left margin on home page.
2. Workaround on ToC positioning by used polyfil [fixed-sticky ] https://github.com/filamentgroup/fixed-sticky) plugin.
3. Some fixes in rollup to be abble to use external plugin from assets.

Fixed-sticky is disabled on mobile devices. On mobile we'll use some another look and position for ToC. Maybe also behaviour should be different such as hidding after tap on link.

**Comment by Reinmar** [comment]
Much better now. Just one problem with lower resolutions, but it seems to be a part of https://github.com/cksource/docs-builder/issues/20 so we don't have to work on it now.

sep-26-2016 10-11-36

screen shot 2016-09-26 at 10 09 09

**Pull request merged**